packages/admin/admin/src/table/AddButton.tsx

```diff
@@ -1,8 +1,8 @@
 import { AddNoCircle } from "@comet/admin-icons";
-import { Button } from "@mui/material";
 import { Component } from "react";
 import { FormattedMessage } from "react-intl";
 
+import { Button } from "../common/buttons/Button";
```

Comment:
This `Button` needs `variant="textDark"`, right? 
Otherwise it will be the new `primary` variant.